lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -159,4 +224,15 @@ void torch_to_tcp::populateTcpCustomOpPatternsAndLegality(
   INSERT_ATEN_TO_TCP_CUSTOM_OP_PATTERN(AtenIndexTensorHackedTwinOp);
   INSERT_ATEN_TO_TCP_CUSTOM_OP_PATTERN(Aten_IndexPutImplOp);
 #undef INSERT_ATEN_TO_TCP_CUSTOM_OP_PATTERN
+
+  auto isNonTransposedConvOp = [](AtenConvolutionOp op) {
+    bool transposed = false;
+    if (matchPattern(op.getTransposed(), m_TorchConstantBool(&transposed)))
+      return !transposed;
+    return false;
+  };
```

Comment:
(minor nit, feel free to push back) It is a bit difficult to interpret from this, so asking here: 
What is the convention for `DynamicLegalityCallbackFn`? In other words can we frame this as `isTransposedConvOp` instead for readability?

Reply:
It is going to be passed to `addDynamicallyLegalOp` which is why we need to define when an op is legal instead of needing to be lowered.

I agree that this means that we have to use a "negated" function because often what we want to specify is what we want to lower instead.

I wish MLIR allowed `addDynamicallyIllegalOp` but I do not see it. I vote to keep it as is for now.